build/config/PolicheckExclusions.xml

```diff
@@ -0,0 +1,11 @@
+<PoliCheckExclusions>
+  <!-- All strings must be UPPER CASE -->
+  <!--Each of these exclusions is a folder name -if \[name]\exists in the file path, it will be skipped -->
+  <Exclusion Type="FolderPathFull">winrt|.git|oss|packages</Exclusion>
+  <!--Each of these exclusions is a folder name -if any folder or file starts with "\[name]", it will be skipped -->
+  <!--<Exclusion Type="FolderPathStart">ABC|XYZ</Exclusion>-->

```

Comment:
This Exclusion tag and the one below are commented out.

Reply:
Yeah those came from the template. I left them here intentionally for future exclusions.